### PR TITLE
feat(Playroom): add icon keywords to playroom scope

### PR DESCRIPTION
**Pull request by marcoskolodny**

### CODE REVIEW

File: src/generated/mistica-icons/icons-keywords.tsx [hunk 1]
```diff
@@ -381,15 +381,15 @@ const iconKeywords: {[key: string]: Array<string>} = {
     ],
     audio: ['sound', 'audio signal', 'soundwave', 'sonido', 'sinal de áudio', 'Ton'],
     aura: ['assistant', 'virtual', 'helper', 'aura', 'aura virtual', 'Helfer'],
-    'authentication-failure': [
+    'autentication-failure': [
```
Comment (marcoskolodny) [inline src/generated/mistica-icons/icons-keywords.tsx:384]: Unrelated change. It's just an update in Mistica design repo

Comment (atabel) [reply src/generated/mistica-icons/icons-keywords.tsx:384]: but it's a typo, isn't it?

**Review by atabel — APPROVED** [review]

File: src/generated/mistica-icons/icons-keywords.tsx (hunk 1 above)
Comment (yceballost) [reply src/generated/mistica-icons/icons-keywords.tsx:384]: we added to 15.0.0 major version to solve this typo in some icons and avoid breaking changes now
https://github.com/Telefonica/mistica-web/issues/626
